plugins/inputs/cisco_telemetry_mdt_netconf_dialin/README.md

```diff
@@ -0,0 +1,105 @@
+# Cisco NETCONF telemetry
+
+Cisco NETCONF telemetry is an input plugin that can consume telemetry data from a [stream of NETCONF notifications](https://tools.ietf.org/html/rfc5277) or from polling. The plugin offers three ways to receive data:
+
+1. Subscription-based, periodic model-driven telemetry streaming through `ietf-yang-push` notifications (see [draft-ietf-netconf-yang-push-25](https://tools.ietf.org/html/draft-ietf-netconf-yang-push-25))
+2. Subscription-based, event notification streaming through `notification`s (see [RFC5277](https://tools.ietf.org/html/rfc5277#page-9))
+3. Periodic polling with `<get>` requests (see [rfc6241 - Network Configuration Protocol (NETCONF), section 7.7](https://tools.ietf.org/html/rfc6241))
+
+The plugin requires the [netgonf library](github.com/cisco-ie/netgonf).
+
+## Sample Configuration
+```bash
```

Comment:
```suggestion
```toml
```

Sample config is in TOML format and should only contain the input config, no output.

Reply:
@NetworkGuy-0x06 updated this file in commit https://github.com/influxdata/telegraf/pull/7392/commits/52087b5832f78211432f0394807a233b0ffeccb8#diff-aa18fdd223c95ce9fd589832ed94570607e9ac70fc0bae4cfcf0a9651f3144a4R13.

Thank you.